package.json

```diff
@@ -19,14 +19,14 @@
     "backbone": "~1.1.2",
     "bootstrap": "3.3.x",
     "bootstrap-switch": "3.3.2",
-    "colors": "0.6.2",
+    "colors": "1.1.2",
     "css-loader": "^0.23.1",
     "eonasdan-bootstrap-datetimepicker": "~4.15",
     "event-source": "0.1.1",
-    "extract-text-webpack-plugin": "2.0.0-beta.5",
-    "file-loader": "^0.x",
+    "extract-text-webpack-plugin": "^2.0.0-rc.0",
+    "file-loader": "^0.9.0",
```

Comment:
Do you want to update all the loaders (`css-loader@0.26.1` and `file-loader@0.10.0`)?  I tried updating them in my debugging adventure, so I think they work.